cmsl1t/analyzers/HW_Emu_jetMet_rates.py

```diff
@@ -115,58 +112,35 @@ def fill_histograms(self, entry, event):
         else:
             maxL1EmuJetEt = 0.
 
-        if nJets == 0:
-            getattr(self, 'singlel1JetEt_rates').fill(pileup, 0.)
-            getattr(self, 'doublel1JetEt_rates').fill(pileup, 0.)
-            getattr(self, 'triplel1JetEt_rates').fill(pileup, 0.)
-            getattr(self, 'quadl1JetEt_rates').fill(pileup, 0.)
-        if nJets >= 1:
-            getattr(self, 'singlel1JetEt_rates').fill(pileup, maxL1JetEt)
-        if nJets >= 2:
-            getattr(self, 'doublel1JetEt_rates').fill(pileup, maxL1JetEt)
-        if nJets >= 3:
-            getattr(self, 'triplel1JetEt_rates').fill(pileup, maxL1JetEt)
-        if nJets >= 4:
-            getattr(self, 'quadl1JetEt_rates').fill(pileup, maxL1JetEt)
-
-        if nEmuJets == 0:
-            getattr(self, 'singlel1JetEt_Emu_rates').fill(pileup, 0.)
-            getattr(self, 'doublel1JetEt_Emu_rates').fill(pileup, 0.)
-            getattr(self, 'triplel1JetEt_Emu_rates').fill(pileup, 0.)
-            getattr(self, 'quadl1JetEt_Emu_rates').fill(pileup, 0.)
-        if nEmuJets >= 1:
-            getattr(self, 'singlel1JetEt_Emu_rates').fill(
-                pileup, maxL1EmuJetEt)
-        if nEmuJets >= 2:
-            getattr(self, 'doublel1JetEt_Emu_rates').fill(
-                pileup, maxL1EmuJetEt)
-        if nEmuJets >= 3:
-            getattr(self, 'triplel1JetEt_Emu_rates').fill(
-                pileup, maxL1EmuJetEt)
-        if nEmuJets >= 4:
-            getattr(self, 'quadl1JetEt_Emu_rates').fill(pileup, maxL1EmuJetEt)
+        for name in self._jetTypes:
+            if 'Emu' in name:
+                getattr(self, name + '_rates').fill(pileup, maxL1EmuJetEt)
+            else:
+                getattr(self, name + '_rates').fill(pileup, maxL1JetEt)
 
         return True
 
+    def _filterByRunAndLumi(self, run, lumi):
```

Comment:
Given the return here is `True` if we want to keep the event and `False` if not, I think this function might be better named something like `_passesLumiFilter`? The current name seems a bit ambiguous to me or, if anything, suggests returning `True` from this function should `filter` out the event (you can't really filter _in_ something).

Reply:
done